src/core/defaults.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Sets the core defaults
+ */
+export const name = "core/defaults";
```

Comment:
I think nearly, if not all, every `name` exported constant is redundant as AFAIK no module uses it.

Reply:
I've kept them only for debugging... when Uglyfied, and ReSpec throws or halts, the core runner prints the name of module as a hint as which module died.

Reply:
https://github.com/w3c/respec/blob/develop/src/core/base-runner.js#L23 for example.

Reply:
Also...
https://github.com/w3c/respec/blob/develop/src/core/base-runner.js#L46

What might be nice, once we have the new class-based plugins, is to build in performance timing directly into the sub class... and error reporting. That will clean up base-runner really nicely. Something for us to discuss elsewhere tho.